lightautoml/transformers/datetime.py

```diff
@@ -243,6 +275,10 @@ def fit(self, dataset: LAMLDataset) -> "LAMLTransformer":
                 self._features.append("season_{0}__{1}".format(s, col))
             if roles[col].country is not None:
                 self._features.append("season_hol__{0}".format(col))
+            if roles[col].last_work_day is not None:
```

Comment:
`last_work_day` is boolean in datetime role, so this `if` statement will always be `True`. Same with `last_day` atribute.